StripeFinancialConnections/StripeFinancialConnections/Source/Common/ExperimentHelper.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ExperimentHelper.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 12/22/22.
+//
+
+import Foundation
+
+/// Abstracts experimentation logic for a specific experiment.
+final class ExperimentHelper {
```

Comment:
I created this to abstract all the logic of experimentation away

I am a little bit going against my rule of only creating a generic class if its helpful in multiple places, but this could be used to simplify `FlowRouter` too; I can do that in a separate PR if it sounds OK